docs/ml/deploy.md

```diff
@@ -34,6 +34,8 @@ For some models, like the [Triton MLModel](https://github.com/viamrobotics/viam-
 {{< relatedcard link="/components/camera/">}}
 {{< /cards >}}
 
+After deploying your model, configure an [`mlmodel` vision service](/ml/vision/) and [`transform` camera](/components/camera/transform/) to visualize the predictions your model makes.
```

Comment:
So the one issue I have with this is that I believe the models are not just for vision services. That's the only thing we currently support natively but it doesn't necessarily have to be a vision service consuming this. 

And the transform camera is entirely optional.

```suggestion
After deploying your model, you need to configure an additional service to use the deployed model.
For example, you can configure an [`mlmodel` vision service](/ml/vision/) and a [`transform` camera](/components/camera/transform/) to visualize the predictions your model makes.
```

Reply:
@npentrel Oh, I did not know that. Do you have an example of what else they're used in?